fdbserver/DataDistribution.actor.cpp

```diff
@@ -660,6 +660,8 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 	PromiseStream<GetMetricsRequest> getShardMetrics;
 	Promise<UID> removeFailedServer;
 
+	Optional<Reference<AsyncVar<struct ServerDBInfo>>> dbInfo;
```

Comment:
I couldn't figure out if we are fetching this and monitoring changes to this somewhere.